common.mk

```diff
@@ -1293,7 +1293,7 @@ update-gems: PHONY
 $(srcdir)/.bundle/gems:
 	$(MKDIR_P) $@
 
-extract-gems: $(srcdir)/.bundle/gems
+extract-gems: update-gems $(srcdir)/.bundle/gems
```

Comment:
Do not download bundled gems when extracting them.

Reply:
Isn't downloading the gems a requirement for extracting them? I had a smarter solution in place that would only download the gems if not already downloaded, but it used a `GNU make` only feature (`shell` function). If I find a way to do that in a platform compatible way, would that be considered a good approach?

Reply:
When extracted from a released (or snapshot) tarball, no more download is needed.
And not only a GNU make feature, `sed` may not be available in that case on Windows.

Reply:
:+1: I see, so essentially, the answer to

>  If I find a way to do that in a platform compatible way, would that be considered a good approach?

is yes, right? Because that would avoid redownloading if the `*.gem` files are already there like in a released tarball.

Reply:
Hmmm, in short, it is OK.

Previously, no additional download was needed once checked out.
Now `configure` tries to download config.guess and config.sub files though.

Reply:
Does `update-gems` no-op if the files are already there? I think that would be convenient and avoid extra connections when building from a release tarball.

Reply:
I'm reading the code now and it seems that it's indeed a no-op if the files are already there, so maybe the current approach could actually work?